vs/dosbox.vcxproj

```diff
@@ -90,7 +90,7 @@
   </ItemDefinitionGroup>
   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">
     <ClCompile>
-      <Optimization>Full</Optimization>
+      <Optimization>MaxSpeed</Optimization>
```

Comment:
Interesting!

Reply:
Took me by surprise as well - in Visual Studio I selected `/O2` as optimization level - which is sort-of like `-O3` in GCC. VS decided to save it in solution this way.